Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -303,14 +301,15 @@ def build_fetch_kwargs(params, occured_start_time, latest_time, search_offset):
         occurred_end_time_fieldname: latest_time,
         "count": FETCH_LIMIT,
         'offset': search_offset,
+        "output_mode": OUTPUT_MODE_JSON,
     }
 
 
 def build_fetch_query(params):
     fetch_query = params['fetchQuery']
 
-    if demisto.get(params, 'extractFields'):
-        extractFields = params['extractFields']
+    if (extract_fields := params.get('extractFields')):
+        extractFields = extract_fields
```

Comment:
a second variable? 🧐  

```suggestion
        extra_raw_arr = extract_fields.split(',')
```